orderer/consensus/hlmirbft/chain.go

```diff
@@ -614,7 +690,24 @@ func (c *Chain) checkMsg(msg *orderer.SubmitRequest) (err error) {
 
 	if msg.LastValidationSeq < seq {
 		c.logger.Warnf("Normal message was validated against %d, although current config seq has advanced (%d)", msg.LastValidationSeq, seq)
+	
+		//JIRA FLY2-103 : Check msg type
+		if c.isConfig(msg.Payload) {
 
+			//JIRA FLY2-103 : get config Metadata from envelope payload
+			configMetaData, err := c.getConfigMetadata(msg.Payload.Payload)
+			if err != nil {
+				return errors.Errorf("bad normal message: %s", err)
+			}
+
+			//JIRA FLY2-103 : get the reconfiguration
+			reconfig,err := c.processReconfiguration(configMetaData)
+			if err != nil {
+				return errors.Errorf("Cannot Generate Reconfiguration Data: %s", err)
+			}
+			//JIRA FLY2-103 : append reconfiguration to c.Node.PendingReconfigurations
+			c.Node.PendingReconfigurations = append(c.Node.PendingReconfigurations, reconfig...)
```

Comment:
One last change. Could you append `reconfig` instead of `reconfig...` and therefore change `PendingReconfigurations` to type `[][]*msgs.Reconfiguration`?